kaspresso/src/main/kotlin/com/kaspersky/kaspresso/testcases/api/testcase/DocLocScreenshotTestCase.kt

```diff
@@ -146,12 +146,12 @@ abstract class DocLocScreenshotTestCase(
                 resourcesDirsProvider,
                 resourceFileNamesProvider
             ),
-            screenshotMaker = CombinedScreenshotMaker(
-                preferredScreenshotMaker = InternalScreenshotMaker(kaspresso.device.activities, screenshotParams),
-                fallbackScreenshotMaker = ExternalScreenshotMaker(
+            screenshotMaker = DocLocScreenshotMaker(
```

Comment:
why?

Reply:
From the begining we used external screenshot maker for DocLoc Test Cases, after some researching i implemented full window screenshots and made it work for DocLoc cases too (in a combined maker), but it's not including system dialog cases, so i implemented new DocLocScreenshotMaker, that will use External maker for "normal" screenshots like it was and Internal maker for full window screenshots to make it work.